bin.src/ingestLinearizeDecam.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, division, print_function
+"""Convert DECam official linearity FITS table into LSST linearity pickles
+
+To use, see UsageStr
+"""
+UsageStr = """Usage: ingestLinearizeDecam.py fromPath repoPath
+    where fromPath is a path to the DECam linearity FITS file
+    and repoPath is the path of the calib data repository
+"""
+import sys
+import os.path
+from itertools import izip
+
+import numpy as np
+import astropy.io.fits as fits
+
+from lsst.daf.persistence import Butler
+from lsst.ip.isr import LinearizeLookupTable
+
+def ingestLinearizeDecam(inPath, repoPath):
+    """Convert the specified DECam linearity FITS table to standard LSST format
+
+    Details:
+    - Input format is one table per CCD, HDU is amplifier number,
+        the table has 3 columns: ADU, ADU_LINEAR_A, ADU_LINEAR_B.
+        The values of ADU contiguous (0, 1, 2...) but check and error out if not.
+        The values of the latter two are replacements (0+delta0, 1+delta1, 2+delta2...)
+        and this is converted to offsets for the LSST linearization tables (delta0, delta1, delta2...)
+    - Output is to %(path)s/linearity/_%02(ccdnum)d.pickle
```

Comment:
In any case it's a bad idea to duplicate information from `DecamMapper.paf` so I changed this to:

    Output is a set of LinearizeLookupTable instances, one per CCD saved as dataset type "linearizer"